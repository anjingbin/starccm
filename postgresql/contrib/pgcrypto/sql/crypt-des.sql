--
-- crypt() and gen_salt(): crypt-des
--

select crypt('', 'NB');

select crypt('foox', 'NB');

create table ctest (data text, res text, salt text);
insert into ctest values ('password', '', '');

update ctest set salt = gen_salt('des');
update ctest set res = crypt(data, salt);
select res = crypt(data, res) as "worked" from ctest;

drop table ctest;

