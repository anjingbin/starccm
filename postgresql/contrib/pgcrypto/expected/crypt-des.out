--
-- crypt() and gen_salt(): crypt-des
--
select crypt('', 'NB');
     crypt     
---------------
 NBPx/38Y48kHg
(1 row)

select crypt('foox', 'NB');
     crypt     
---------------
 NB53EGGqrrb5E
(1 row)

create table ctest (data text, res text, salt text);
insert into ctest values ('password', '', '');
update ctest set salt = gen_salt('des');
update ctest set res = crypt(data, salt);
select res = crypt(data, res) as "worked" from ctest;
 worked 
--------
 t
(1 row)

drop table ctest;
